virtcontainers/qemu_amd64.go

```diff
@@ -20,7 +20,7 @@ const defaultQemuPath = "/usr/bin/qemu-system-x86_64"
 
 const defaultQemuMachineType = QemuPC
 
-const defaultQemuMachineOptions = "accel=kvm,kernel_irqchip,nvdimm"
+const defaultQemuMachineOptions = "accel=kvm,kernel_irqchip,nvdimm,nostatic_prt"
```

Comment:
@jcvenegas why do you define `nostatic_prt` as a default option for `pc` machine type? I thought you said this was only introduced by the very specific qemu-lite branch of qemu. What if we use a vanilla qemu?
I just want to make sure we won't get an error from a vanilla Qemu because we're using a non defined flag by default here.